src/main/java/com/artipie/ArtipieProperties.java

```diff
@@ -54,6 +54,21 @@ public String version() {
         return this.properties.getProperty(ArtipieProperties.VERSION_KEY);
     }
 
+    /**
+     * Obtains timeout of caching configuration files in milliseconds.
+     * @return Timeout of caching files with configuration.
+     */
+    public int configCacheTimeout() {
+        int time;
+        try {
+            time = Integer.parseInt(this.properties.getProperty("artipie.config.cache.timeout"));
+        } catch (final NumberFormatException exc) {
+            // @checkstyle MagicNumberCheck (1 line)
+            time = 2 * 60 * 1000;
```

Comment:
@g4s8 I've replaced with `Integer.getInteger("artipie.config.cache.timeout", 2 * 60 * 1000)` although it does not throw any exception in case of mistyping in `artipie.properties` and I had to set property value at first